ToolBox/QuantConnect.ToolBox.csproj

```diff
@@ -45,6 +45,9 @@
     <Reference Include="EngineIoClientDotNet, Version=0.9.22.0, Culture=neutral, processorArchitecture=MSIL">
       <HintPath>..\packages\EngineIoClientDotNet.0.9.22\lib\net45\EngineIoClientDotNet.dll</HintPath>
     </Reference>
+    <Reference Include="ICSharpCode.SharpZipLib, Version=0.86.0.518, Culture=neutral, PublicKeyToken=1b03e6acf1164f73, processorArchitecture=MSIL">
+      <HintPath>..\packages\SharpZipLib.0.86.0\lib\20\ICSharpCode.SharpZipLib.dll</HintPath>
```

Comment:
`QuantConnect.Compression` references a different version of this library. Please try using the same library and confirm that it works. Having multiple versions of packages quickly devolves into a modern form of dll-hell.
``` xml
<package id="ICSharpCode.SharpZipLib.dll" version="0.85.4.369" targetFramework="net45" />
```